### PR TITLE
feat: adding ExpansionPanel component in bank details for Argentinian users, and also removing ARS currency from the default currencies (this is only for Argentina)

### PR DESCRIPTION
**Pull request by lsena-bc**
## Description
Adding show/hide functionality to the Argentinian bank details, and also the ARS currency was removed from the default currencies because this feature is only for Argentina.

Screen Shot 2022-05-26 at 13 29 40

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  return (
+    <div>
+      {isToggled ? <div>{children}</div> : ''}
+      <Button
+        nature='white-blue'
+        small
+        style={{ margin: '8px auto 0px' }}
+        data-e2e={isToggled ? 'showLess' : 'showMore'}
+        // @ts-ignore
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx:22]: What is the ts error this is ignoring?

File: packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+export const ExpansionPanel: ExpansionPanelComponent = ({ children }) => {
+  const [isToggled, handleToggle] = useState(false)
+
+  return (
+    <div>
+      {isToggled ? <div>{children}</div> : ''}
+      <Button
+        nature='white-blue'
+        small
+        style={{ margin: '8px auto 0px' }}
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx:20]: Could you remove this style prop and wrap the button in a `<Padding></Padding>` component? Here's some instructions https://blockchain.github.io/blockchain-wallet-v4-frontend/?path=/story/structural-padding-intro--page

Comment (lsena-bc) [reply packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx:20]: Thank you Dylan!! Could you re check it please?

**Comment by lsena-bc** [comment]
> i would use more destructing on the props as well as create variables for some as they are accessed multiple times. this way you avoid the super long and repeated object prop drilling
> 
> for example, make a variable out of the account address since its used 5+ times and that way you only have to cast it once
> 
> `const accountAddress = props.account.agent as AgentType).address`

Done, thank you for the feedback @schnogz !

File: packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const [isToggled, handleToggle] = useState(false)
+
+  return (
+    <>
+      {isToggled ? <div>{children}</div> : ''}
+      <Padding top={10}>
+        <Button
+          nature='white-blue'
+          small
+          style={{ margin: '8px auto 0px' }}
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/ExpansionPanel/ExpansionPanel.tsx:22]: can remove this line now since you added the padding?

**Review by blockdylanb — DISMISSED** [review]

**Review by schnogz — APPROVED** [review]

**Pull request merged**